feature-utils/poly-look/test/react-components/contexts/mocks/data-importers-mock.js

```diff
@@ -0,0 +1,6 @@
+class MockImporterClass {
```

Comment:
It's a bit suspicious to call a class "class" - naming-wise we already know it's a class because it's in upper camel case.